site/pages/search-results-contextual-en.hbs

```diff
@@ -0,0 +1,104 @@
+<!--Note 2: this template should only be used for contextual search results from a single institution-->
+<!--Note 3: until radio buttons pattern works, please simply include 'All government of Canada websites' link in the well, under the search box-->
+<div class="col-md-12">
+	<h1 property="name">Search results</h1>
+	<div class="well">
+		<form name="cse-search-box" method="GET" action="/en/sr/srb.html" id="cse-search-box">
+			<label class="wb-inv" for="sch-inp">Search Government of Canada websites</label>
+			<input type="hidden" name="st" value="s" />
+			<input type="hidden" name="s5bm3ts21rch" value="x" />
+			<input type="hidden" name="num" value="10" />
+			<input type="hidden" name="st1rt" value="0" />
+			<input type="hidden" name="langs" value="eng" />
+			<input type="hidden" name="cdn" value="canada" />
+			<input type="hidden" name="hq" value="" />
+			<div class="input-group mrgn-tp-lg">
+				<input value="[value]" spellcheck="false" autocomplete="off" dir="ltr" id="sch-inp" name="q" class="form-control" type="search">
+				<span class="input-group-btn">
+					<button type="submit" class="btn btn-primary btn-small" name="wb-srch-sub"> <span class="glyphicon-search glyphicon"></span><span class="wb-inv">Search</span></button>
+				</span>
+			</div>
+		</form>
+		<p class="wb-inv">See results from:</p>
+		<ul class="inline">
+			<li><input type="radio" name="searchscope" id="radio-allresults" value="allresults"  /> <label id="label-alldepartments">All Government of Canada websites</label></li>
+			<li><input type="radio" name="searchscope" id="radio-context" value="context" checked="checked" /> <label id="label-singledepartment"><strong>[context label]</strong></label></li>
+		</ul>
+	</div>
+</div>
+<div class="col-xs-12">
+	<h2>[number] search results for "[search term]"</h2>
+	<p class="gc-byline"><strong>From [Institution name]</strong></p>
+	<!--Note: byline is only included in contextual versions of the results page-->
+	<section>
+		<h3><a href="#">[Search result page title (not including '- Canada.ca' or other institutional suffix)]</a></h3>
+		<ul class="label">
+			<li>[institution name]</li>
+			<li>[institution name]</li>
+			<li>[institution name]</li>
+			<!--Note: if labels go past 1 row, then option a) cut off at end of row, option b) do not show any labels beyond the first row. Text-overflow of the cut off label should be with an ellipsis. Expand/collapse to show all is not a requirement for Jan 2018 MVP-->
+			<!--Note 2: First label should always be the label of the context the search is in (if contextualised). Not necessarily order of tags in page metadata.-->
+		</ul>
+		<ol class="breadcrumb">
+			<li>[domain name of search result]</li>
+			<li>[parent breadcrumb of page]</li>
+			<!--Note: this always has 2 list items. Does not include other breadcrumb items.-->
+		</ol>
+		<!--when the breadcrumb is not available to the search engine, use the URL instead-->
+		<url>[url of the search result]</url>
```

Comment:
```<url>``` is not a valid element. You can use ```<p class="url">``` or ```<p><cite><a href="[link]">[link]</a></cite></p>```
Using the ```<cite>``` element is might be the best option as per it's definition in the HTML 5 spec: https://www.w3.org/TR/html53/textlevel-semantics.html#the-cite-element

Reply:
K, I'll add that, and should pass Travis, finally! Thanks Pierre!